src/test/config/jira/helm_parameters

```diff
@@ -2,6 +2,7 @@ RELEASE_PREFIX=${helm.release.prefix}
 CHART_SRC_PATH=${helm.src.path}/jira
 PRODUCT_CHART_VALUES_FILES=${project.build.directory}/config/jira/values
 PRODUCT_INGRESS_TEMPLATE=${project.build.directory}/config/jira/ingress
+PRODUCT_SERVICE_EXPOSE_NODE_TEMPLATE=${project.build.directory}/config/jira/service-expose-node
```

Comment:
We'll need these changes made to all 3 products' `helm_parameters` files

Reply:
Do we? Other products won't use it unless they specify `TARGET_REPLICA_COUNT`. If `TARGET_REPLICA_COUNT` is undef, this won't be used.